lib/features/welcome/views/welcome_page.dart

```diff
@@ -1,68 +1,70 @@
 import 'package:flutter/material.dart';
-import 'package:kahootify/features/welcome/views/widgets/fancy_button.dart';
+import 'package:kahootify/color_consts.dart';
+import 'package:kahootify/features/settings/views/settings_page.dart';
+import 'package:kahootify/features/welcome/views/widgets/welcome_button.dart';
 
 class WelcomePage extends StatefulWidget {
   @override
   _WelcomePageState createState() => _WelcomePageState();
 }
 
 class _WelcomePageState extends State<WelcomePage> {
-  String value = '10';
 
-  void changeToTwenty(BuildContext superContext) {
-    print("test");
-    showDialog(
-        context: superContext,
-        builder: (context) {
-          return Material(
-            child: Center(
-              child: Container(
-                child: Column(
-                  children: [
-                    Text("cześć"),
-                    ElevatedButton(
-                      onPressed: () {
-                        setState(() {
-                          value = "123";
-                        });
-                        Navigator.of(context).pop();
-                      },
-                      child: Text("Wyjście"),
-                    ),
-                  ],
-                ),
-              ),
-            ),
-          );
-        });
-    setState(() {
-      value = (int.parse(value) + 1).toString();
-    });
-  }
+
 
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      backgroundColor: kBackground2Color,
       appBar: AppBar(
-        title: Text("WELCOME SCREEN"),
+        title: Text("KAHOOTIFY.LY"),
+        backgroundColor: kBackground1Color,
+        actions: <Widget>[
+          Padding(
+            padding: EdgeInsets.only(right: 20.0),
+            child: GestureDetector(
```

Comment:
Instead of using Gesture detector with Icon as child, use IconButton for simple icon buttons like this one! Icon button apart from easier use provides neat animations when pressed

Reply:
Ready